src/data/finset/basic.lean

```diff
@@ -277,7 +277,7 @@ protected def empty : finset α := ⟨0, nodup_zero⟩
 
 instance : has_emptyc (finset α) := ⟨finset.empty⟩
 
-instance : inhabited (finset α) := ⟨∅⟩
+instance inhabited_finset : inhabited (finset α) := ⟨∅⟩
```

Comment:
I am surprised that I needed to do something here: Lean is not able to choose an instance name by itself. Could it be fixed upstream?

Reply:
This might be related due to the `def _root_.foo :=` change.  Maybe because it's an instance definition in a namespace?  But I have no idea why it works for the `has_emptyc` instance above, then.

In any case, it only seems to concern two instances.  I can take a look at it at some point.  Preparing a mathlib-free MWE and filing an issue at the lean bug tracker would help.